XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs

```diff
@@ -95,88 +95,158 @@ internal static OVRControllerBase[] Controllers
         internal static Controller activeControllerType = Controller.None;
         internal static Controller connectedControllerTypes = Controller.None;
 
+        public static bool Initialized
+        {
+            get
+            {
+                return ovrp_GetInitialized() == Bool.True;
```

Comment:
These could use the cleaner syntax to help reduce the number of lines. (Same with all the things in this class)

```
public static bool Initializeed
{
    get => ovrp_GetInitialized() == Bool.True;
}
```

Reply:
Oculus API code, can't change to retain license